content/en/blog/interoperability/index.md

```diff
@@ -0,0 +1,76 @@
+
+{{< figure src="https://courses.edx.org/assets/courseware/v1/571727dd6d3f57d64158c9567f0d8ff2/asset-v1:LinuxFoundationX+LFS173x+1T2020+type@asset+block/The_Aries_Agent_Test_Harness.png" attr="image source: LinuxFoundationX LFS173x (CC BY 4.0)" attrlink="https://learning.edx.org/course/course-v1:LinuxFoundationX+LFS173x+1T2020/home" >}}
+
+Harness utilizes Docker containers for testing. It launches a container based on a required agent image for each test scenario actor during the test run. Before the test run, one needs to build and bundle the agent-required services and the backchannel to a single agent image that provides both the agent and the backchannel functionality. The recipes for making each of the different agent images, i.e., Dockerfiles with the needed scripts, are stored in the AATH repository. The same repository also contains CI scripts for executing the tests regularly and generating [an extensive test report site](https://aries-interop.info/).
+
+## Interoperability for Findy Agency
+
+One of our main themes for 2H/2021 was to verify the Aries interoperability level for Findy Agency. When I investigated the Aries interoperability tooling more, it became evident that we needed to utilize the AATH to accomplish the satisfactory test automation level.
+
+My first task was to create [a backchannel](https://github.com/findy-network/findy-agent-backchannel) for the harness to operate Findy Agency-hosted agents. Backchannel's role is to convert the harness's REST API requests to Findy Agency gRPC client interface. Another challenge was to bundle the agency microservices into a single Docker image. Each agency microservice runs in its dedicated container in a regular agency deployment. For AATH, I needed to bundle all of the required services into a single container, together with the backchannel. [The bundle](https://github.com/findy-network/findy-agent-backchannel/blob/master/aath/Dockerfile) is necessary because the harness runs a single container per agent.
+
+Once the bundle was ready, I made [a PR to the AATH repository](https://github.com/hyperledger/aries-agent-test-harness/pull/341) to include Findy Agency in the Aries interoperability test set. We defined the supported AIP version (1.0) and the exception features (revocation). Tests exposed some essential but mainly minor interoperability issues with our implementation, and we were able to solve all of the found problems quite swiftly. The tests use the latest Findy Agency release with each test run. One can monitor [the test results for Findy Agency](https://aries-interop.info/findy.html) on the test result site.
```

Comment:
"We defined the supported AIP version (1.0) and the exception features (revocation)."
Difficult to read and understand. Maybe:
We decided to support AIP version 1.0, but leave out the revocation for now.